### PR TITLE
Feature/login component

### PR DESCRIPTION
**Pull request by Venomaleks**
[ch]
## Description
Description of your changes
This page can be used for login or sign up. 
## Checklist
- [ ] At the top of this comment add the clubhouse story number e.g. \`[ch123]\`
- [x] Does the branch adhere to our branch naming conventions?
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] Have you successfully ran tests with your changes locally?
- [ ] All new and existing tests passed.
- [ ] Have you added an explanation of what your changes do?
## How to test/verify this feature
Explain how in an ordered list of whats required to test this feature. 
1. Do a 'npm start' in your terminal white the project open.
2. When you are in the home page, press on the sign up button.
3. When it opens the new page type in the email and password.
4. Select one of the radio checkboxes, se if only one can be checkt.
5. then se if you can se if the button can be presset.

### CODE REVIEW

File: src/components/AboutPage/AboutSubCard/AboutSubCard.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import { Container } from "react-bootstrap";
+
+const AboutSubCard = () => {
+  return(
+  <div>
+    <Container>
+    <SubBoxes>  
+      <SubBox>
+          <SubTitel>Subscribe</SubTitel>
```
Comment (Minnia) [inline src/components/AboutPage/AboutSubCard/AboutSubCard.tsx]: Gör om namnet från "SubTitel" till "SubTitle"

File: src/components/AboutPage/AboutSubCard/AboutSubCard.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          <SubText>For what users:info</SubText>
+        
+      <hr/>
+        
+        <SubText>for haw long</SubText>
+        
+      <hr/>
+        
+        <SubText>
+          <SubButton>sub naw</SubButton>
```
Comment (Minnia) [inline src/components/AboutPage/AboutSubCard/AboutSubCard.tsx:35]: Vad betyder "sub naw"?

File: src/components/AboutPage/AboutSubCard/styled.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+border-radius: 25px;
+justify-content: space-evenly;
+margin: 5%;
+`;
+
+export const SubRows = styled(Row)`
+border: 5px solid gray;
+`;
+
+export const SubTitel = styled.h2`
```
Comment (Minnia) [inline src/components/AboutPage/AboutSubCard/styled.ts]: Döp om till "SubTitle"

File: src/components/SignupPage/SignupCustomer/SignupCustomer.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ import {
   SignupButton,
   ComponentContaner,
   RowObjects,
-  Logindiv,
+  Signdiv,
```
Comment (Minnia) [inline src/components/SignupPage/SignupCustomer/SignupCustomer.tsx:9]: Istället för Signdiv borde det vara SignDiv, då det är två ord

File: src/components/AboutPage/AboutSubCard/AboutSubCard.tsx (hunk 2 above)
Comment (Venomaleks) [reply src/components/AboutPage/AboutSubCard/AboutSubCard.tsx:35]: Subscribe now

File: src/components/AboutPage/AboutSubCard/styled.ts (hunk 3 above)
Comment (Venomaleks) [reply src/components/AboutPage/AboutSubCard/styled.ts]: yes, jag kan göra det. :)

File: src/components/SignupPage/SignupCustomer/SignupCustomer.tsx (hunk 4 above)
Comment (Venomaleks) [reply src/components/SignupPage/SignupCustomer/SignupCustomer.tsx:9]: det låter logiskt kan fixa det.

**Pull request merged**